client/components/app-promo/test/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'AppPromo', ( ) => {
+	useFakeDom();
+
+	const appPromoDetails = {
+		promoCode: 'a0001',
+		message: 'WordPress.com your way  — desktop app now available for Mac, Windows, and Linux.'
+	};
+	const appPromoLink = 'http://www.wordpress.com';
+
+	let AppPromo;
+	let AppPromoComponent;
+	const defaultPropsToRender = {
+		location: 'reader',
+		promoItem: appPromoDetails,
+		getPromoLink: () => appPromoLink
+	};
+	before( ( ) => {
+		AppPromo = require( '..' ).AppPromo;
+		AppPromoComponent = ( <AppPromo { ...defaultPropsToRender } /> );
+	} );
```

Comment:
The `useFakeDom()` needs to kick in before I can load up the `AppPromo` component without bumping into an error about document not being found.